messaging/methods/media/listMedia.md

```diff
@@ -174,4 +181,30 @@ while True:
     if continuation_token is None:
         break
 ```
+
+{% sample lang="js" %}
+
+```js
+async function getMediaWithToken(continuationToken) {
+    await messagingController.listMedia(userId, continuationToken, function(error, response, context) {
+        console.log("Medias length: " + response.length);
+        console.log("Media 1 name: " + response[0].mediaName);
+        if (context.response.headers.hasOwnProperty('continuation-token')) {
+            continuationToken = context.response.headers['continuation-token'];
+        } else {
+            continuationToken = null;
+        }
+    });
+    return continuationToken;
+}
+
+var continuationToken = '';
+while (true) {
```

Comment:
Why not 
```
var continuationToken = true; //Also could be a string like "continuation token"
while( continuationToken ) {
     continuationToken = await getMediaWithToken(continuationToken);
}

Reply:
Unfortunately that doesn't work because passing `true` to `messagingController.listMedia(userId, continuationToken, function(error, response, context) {})` (which is what is eventually called) causes the request to our API to fail. This example is to show how to conditionally continue to retrieve continuation tokens starting with no token

Reply:
This is also a symptom of the sdks not handling optional parameters